### ISSUE TITLE
[CAIA A11y Research]: govcio-vfep, GI Bill Comparison Tool, Specialized Mission Filter (September 2023)

### ISSUE THREAD

**Issue by arianaadili** [department-of-veterans-affairs/va.gov-team#65106]
## CAIA Assignees
**Support goal:** assist users with assistive technology, should any issues take place, during the sessions. Identify and document any accessibility-related issues encountered in the user testing sessions. 

- [x] Sara
- [x] SK
- [x] Jamie
- [x] **`RCVD 9/20/2023` session times for dates provided:** September 25 - 29, 2023. 
_There are currently three studies overlapping during this timeframe, and we will do our very best to provide support for your AT sessions._
- [x] Added date range to the [CAIA Capacity Calendar](https://calendar.google.com/calendar/embed?height=600&wkst=1&bgcolor=%23ffffff&ctz=America%2FNew_York&mode=WEEK&src=Y19mOTgyMzUwZmM3M2E4MTYyYTEwODhhYjE0ZDAwZGQzNzMzNWIyYjU1OWE2MGQ2YjY3ZWI4MDIxNDE2ZjgxYzJhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20&color=%23616161).

### Who's on your team?

```bash
Team name: govcio-vfep
OCTO-DE product owner: Emily Theis (emily.theis@va.gov)
Product manager: Kara Ciprich (kara.ciprich@va.gov)
Designer: Ariana Adili (ariana.adili@govcio.com / ariana.adili@va.gov)
FE engineer: TBD
Product/team Slack channel: #govcio-vfep-team 
https://join.slack.com/share/enQtNTg1Mjg4NTI0MTEwOC1kZmIzMjM4ZTMzMGM0NGZmM2EyZjZlMzI0NmNhNGZhNmRmMTVlOTUzNWU1ZDlmNTYwOTFmNzIxNzkxMzY2ZTVk
...
```

### Do you have an accessibility champion?

No, we don't have a champion (yet)

### Calendar checking

- [X] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [X] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [X] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

### When is the timeframe for your testing?

9/25 - 9/29

### Supporting artifacts (if any)

- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/Comparison%20Tool%20Enhancements/GI%20Bill%20Comparison%20Tool_Specialized%20Mission%20Filter%20Product%20Product%20Outline_v2.docx)
- [User flows](Insert URL here between the parenthesis)
- [Prototype or mockups](https://sketch.com/s/3edef2a8-47eb-456d-bfec-60da90ae51a1) _Page: Search by name tab_ | [Link to design in staging](https://staging.va.gov/education/gi-bill-comparison-tool)
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/Comparison%20Tool%20Enhancements/research/Research_Plan_CT_SMF.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/Comparison%20Tool%20Enhancements/research/Conversation_Guide_CT_SMF.md)
- [Epics/issue that may be helpful](https://app.zenhub.com/workspaces/sitewide-content-accessibility-and-ia-63a1d63232beba0011a7833f/issues/gh/department-of-veterans-affairs/va.gov-team/64172) & [Link to collaboration cycle ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57486)
...

### Next steps

- [x] Post a link to this ticket in the accessibility-help Slack channel and tag Terry Nichols.
- [x] Your team's researchers and observers read through platform guidance on research with assistive technology users
- [x] CAIA Specialists are assigned to your research, be sure to add the respective a11y specialist(s) to your session invites (include the Zoom or Teams link).
- [x] CAIA Specialists review your conversation guide and research plan
- [x] **Provide specialists your participant details like skill level, operating system, assistive tech used, browser** @arianaadili 
- [ ] CAIA specialists will provide you an optional feedback form to let us know how we can support y'all better into the future.
- [ ] CAIA Specialists create a research notes template
- [ ] CAIA Post research, specialists will add accessibility findings to your research folder

**Comment by sara-amanda** [comment]
## CAIA Update: Research Session Support (Pending Session Dates)
@arianaadili thank you for submitting the research support request! 

When you have the AT research sessions, feel free to tag me and @coforma-terry, so we can make sure we can join as many as possible during. your 9/25 - 9/29 testing.

**Comment by arianaadili** [comment]
> ## CAIA Update: Research Session Support (Pending Session Dates)
> @arianaadili thank you for submitting the research support request!
> 
> When you have the AT research sessions, feel free to tag me and @coforma-terry, so we can make sure we can join as many as possible during. your 9/25 - 9/29 testing.

Thank you @sara-amanda, will do! Also just wanted to mention that I updated the link to my research plan on this ticket as I've made a few updates to it. Just wanted to let you know

**Comment by arianaadili** [comment]
Hi @sara-amanda & @coforma-terry, our usability tests have been scheduled for the week of 9/25-9/29. I just have to provide Perigean with your emails in order for you to receive invites to the sessions with AT users. Could you please provide your emails so I can ensure that you receive these invites? Thank you!

**Comment by coforma-terry** [comment]
Hi @arianaadili - please use the following:
sara.smith@oddball.io
sarah.koomson@oddball.io

We can share and add more later :)
Thanks!

**Comment by sara-amanda** [comment]
## CAIA AT SESSIONS
### GI Bill Comparison Tool, Specialized Mission Filter
- [x] Invites received for the following four dates: 
   - Monday 9/25, 11 a.m. - noon
       - @sara-amanda available.
   - Thursday 9/28, 11:30 a.m. -noon
        - @sara-amanda available.
    - Thursday 9/28, 2:30-3 p.m.
        - `CAIA Open Slot`
        - @sara-amanda (unavailable)
    - Friday, Sept. 29, 11:30 a.m. - noon
       - @sara-amanda available.
- [ ] Will there be any additional invites @arianaadili or is this the full set? 
- [x] Blocks placed on the CAIA Capacity Calendar @coforma-terry

**Comment by sara-amanda** [comment]
@arianaadili thank you for the quick update, if you could add the  additional info needed, next to each slot we would be most grateful.

**Comment by arianaadili** [comment]
> > ## CAIA AT SESSIONS
> > ### GI Bill Comparison Tool, Specialized Mission Filter
> > 
> > * [x]   Invites received for the following four dates:
> >   
> >   * Monday 9/25, 11 a.m. - noon
> >     
> >     * @sara-amanda available.
> >   * Thursday 9/28, 11:30 a.m. -noon
> >     
> >     * @sara-amanda available.
> >   * Thursday 9/28, 2:30-3 p.m.
> >     
> >     * `CAIA Open Slot`
> >     * @sara-amanda (unavailable)
> >   * Friday, Sept. 29, 11:30 a.m. - noon
> >     
> >     * @sara-amanda available.
> > * [x]   Will there be any additional invites @arianaadili or is this the full set?
> >   @sara-amanda This is the full set. We will only have 3 AT participants in our study (ChromeVox, VoiceOver, and JAWS).
> > * [x]   Blocks placed on the CAIA Capacity Calendar @coforma-terry
> > 
> > ## Additional Information Needed
> > **To best serve the users of VA.gov, we will need to collect the following information during each a11y research session:**
> > 
> > * Participant: 1
> > * Participant Testing Date/Time: 9/25, 11 AM - 12 PM EST
> > * Skill level of user: Unsure
> > * Operating System: Unsure
> > * Assistive Technology being used: ChromeVox
> > * Internet Browser: Unsure, most likely Chrome
> > * Device Type: Laptop

> > * Participant: 12
> > * Participant Testing Date/Time: 9/28, 11:30 AM - 12 PM EST
> > * Skill level of user: Unsure
> > * Operating System: Unsure
> > * Assistive Technology being used: VoiceOver
> > * Internet Browser: Unsure, most likely Chrome
> > * Device Type: Laptop

> > * Participant: 13
> > * Participant Testing Date/Time: 9/28, 2:30 - 3:00 PM EST
> > * Skill level of user: Unsure
> > * Operating System: Unsure
> > * Assistive Technology being used: JAWS
> > * Internet Browser: Unsure, most likely Chrome
> > * Device Type: Laptop

* Sorry I am not able to provide accurate answers for their skill levels, operating systems, and their internet browsers. I was not provided this information. 

> > 
> > @SarahKay8

**Comment by SarahKay8** [comment]
Hey @arianaadili! Please see my available dates. Meeting invites have been accepted.

Monday 9/25, 11 a.m. - noon
@SarahKay8 available.

Thursday 9/28, 2:30-3 p.m.
@SarahKay8 available
Friday, Sept. 29, 11:30 a.m. - noon
@SarahKay8  available.

cc: @sara-amanda

**Comment by arianaadili** [comment]
> Hey @arianaadili! Please see my available dates. Meeting invites have been accepted.
> 
> Monday 9/25, 11 a.m. - noon @SarahKay8 available.
> 
> Thursday 9/28, 2:30-3 p.m. @SarahKay8 available Friday, Sept. 29, 11:30 a.m. - noon @SarahKay8 available.
> 
> cc: @sara-amanda

Hey @SarahKay8 thanks! I just made an update to my comment above. I completely missed the fourth AT participant from our list. Looks like you have availability at that time for this session. Just wanted to point this out. Thanks again!

**Comment by sara-amanda** [comment]
## CAIA AT Sessions

### Participant: 1
#### Participant Testing Date/Time: 9/25, 11 AM - 12 PM EST
- **CAIA a11y:** @sara-amanda 
- **CAIA a11y:** @SarahKay8 
- Skill level of user: Unsure
- Operating System: Unsure
- Assistive Technology being used: ChromeVox
- Internet Browser: Unsure, most likely Chrome
- Device Type: Laptop

### Participant: 12
#### Participant Testing Date/Time: 9/28, 11:30 AM - 12 PM EST
- **CAIA a11y:** @sara-amanda 
- Skill level of user: Unsure
- Operating System: Unsure
- Assistive Technology being used: VoiceOver
- Internet Browser: Unsure, most likely Chrome
- Device Type: Laptop

### Participant: 13
#### Participant Testing Date/Time: 9/28, 2:30 - 3:00 PM EST
- **CAIA a11y:** @SarahKay8 
- Skill level of user: Unsure
- Operating System: Unsure
- Assistive Technology being used: JAWS
- Internet Browser: Unsure, most likely Chrome
- Device Type: Laptop

### Participant: 15
#### Participant Testing Date/Time: 9/29, 11:30 - 12:00 PM EST
- **CAIA a11y:** @sara-amanda 
- **CAIA a11y:** @SarahKay8 
- Skill level of user: Unsure
- Operating System: Unsure
- Assistive Technology being used: VoiceOver
- Internet Browser: Unsure, most likely Chrome
- Device Type: Desktop

**Note**: _There are currently ~three~ two studies overlapping during this timeframe (including yours), and we will do our very best to provide support for your AT sessions. This is why you will see more than one CAIA member on a date listed above._

cc: @SarahKay8 @coforma-terry @arianaadili

**Comment by sara-amanda** [comment]
## CAIA Update
@arianaadili prior to the user research sessions being held today, @coforma-jamie was reviewing the staging links and noticed an issue we want to bring to your attention, as it impacts users of assistive technology, specifically Screen Readers. 

### Fieldset with No Legend

- The “specialized mission” section is a `fieldset` with no `legend` …
-  So the first `div` within the `fieldset` is being treated as the legend (in VoiceOver, anyways)…
- When you get to the checkboxes, it reads as "Specialized mission Jump to specialized mission details Historically Black college or university, checkbox"

🧵 [Thread started in Slack channel: #govcio-vfep-team](https://dsva.slack.com/archives/C04D5FS91BK/p1695911660320649)

@SarahKay8 @coforma-terry @coforma-jamie

**Comment by SarahKay8** [comment]
![DC63F99D-D187-43A8-87F9-90D5D2B064AA](https://github.com/department-of-veterans-affairs/va.gov-team/assets/102324990/8894b82a-8507-4235-80f9-8a0a049aaf7a)
"About this tool" link opens in new window/tab and does not warn user. Opening new windows automatically when a link is activated can be disorienting for people who have difficulty perceiving visual content, and for some people with cognitive disabilities, if they are not warned in advance. Providing a warning allows the user to decide it they want to leave the current window, and the warning will help them find their way back, if they do decide they would like to go to the new window. It will help them understand that the "back" button will not work and that they have to return to the last window they had open, in order to find their previous location.
@coforma-jamie

**Comment by coforma-jamie** [comment]
The only actionable findings came from P15, summarized below:

- _What would you do if you were looking for a school?_ “I would search by location and type in my location zip code. I would narrow down these filter options by typing in more specific keywords”
- In “Military Status” field she advised content should be changed from “child” to “eligible dependent”
- _Where would you navigate to find “results”?_ I would navigate to the update results.
- “When I click on the update results button, I presume it would change the filters at the top of the form. It’s confusing.” I would get rid of the “Filter by Location” card.

Our other feedback can be found in the comments above.

@arianaadili, please reach out if you have any questions! Thank you :) 

cc @sara-amanda
